app/controllers/dashboard_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class DashboardController < ApplicationController
-  before_action :verify_developer, except: [:index, :new_dash, :spam_by_site]
+  before_action :verify_developer, except: [:index, :new_dash, :spam_by_site, :site_dash]
```

Comment:
It would probably make more sense to change this to `before_action :verify_developer, only: [:db_dumps, :download_dump]`.